apps/user-office-backend/db_patches/0130_AddTimedLogoutWarningFeature.sql

```diff
@@ -0,0 +1,12 @@
+DO
+$$
+BEGIN
+    IF register_patch('AddTimedLogoutWarningfeature.sql', 'Thomas Cottee Meldrum', 'Create STFC idle warning feature', '2023-02-14') THEN
+
+    INSERT INTO features(feature_id, description) VALUES ('STFC_IDLE_TIMER', 'STFC idle warning popup');
+    INSERT INTO settings(settings_id, description) VALUES ('IDLE_TIMEOUT', 'Timour for Idle timer');
```

Comment:
I may be wrong but is it meant to be `Timour` there?

Reply:
Yeah typo thanks for spotting that